rides-android/src/test/java/com/uber/sdk/android/rides/RobolectricTestBase.java

```diff
@@ -30,6 +30,6 @@
  * Base test class to remove use of annotations on all test classes.
  */
 @RunWith(RobolectricTestRunner.class)
-@Config(sdk = 21)
+@Config(sdk = 26)
```

Comment:
same q on unintended consequences. Will this change our internal test base default SDK ver?

Reply:
no, this is just in the 3P repository. we don't consume it internally